lib/posts/20170112_mongodb_graphlookup.md

```diff
@@ -165,7 +165,7 @@ db.data.aggregate([
       connectFromField: 'actors', // Match actors in one movie...
       connectToField: 'actors', // to actors in another movie
       as: 'connections',
-      // Add a 'steps' property to each connections subdoc that contains
+      // Add a 'steps' property to each connections' subdoc that contains
```

Comment:
I don't think the apostrophe belongs here. Maybe "each connection subdoc" instead?

Reply:
i don't know, I'm not a native speaker :) maybe i compensate by adding apostrophes when is not needed :)